style.css

```diff
@@ -781,3 +781,170 @@ p {
 .input1 > label {
   display: none;
 }
+
+/* Mobile Popup */
+
+.popUp {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 24px;
+  position: fixed;
+  width: 343px;
+  height: 620px;
+  left: calc(50% - 343px / 2);
+  top: 30px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+}
```

Comment:
- [ ] You have done great so far but there are still some changes you need to implement before moving to the next milestone. :+1: :100: 
- [ ] As mentioned by the previous reviewer, your pop-up does not fully match the Figma design. Looking at the screenshot below, you can see that the image is out of the pop-up. I'll suggest you fix this. :+1: 

- - Screenshot:
![Screenshot from 2023-06-21 20-01-23](https://github.com/sulaimanahmadi/Portfolio/assets/107632780/a9b34dc7-2681-437b-8fa7-593668f6afe0)

- [ ] Also for the mobile screen above `767 px`, the pop-up does not look great. The text tends to over flow the pop on the bottom. I'll suggest you take a look at this also. :+1: :smile: 

- - Screenshot:
![Screenshot from 2023-06-21 20-03-06](https://github.com/sulaimanahmadi/Portfolio/assets/107632780/648223dd-1f15-49ba-961b-066b8e9fc734)